### ISSUE TITLE
ResNet-RS: Was dropblock used during training?

### ISSUE THREAD

**Issue by xhlulu** [tensorflow/tpu#904]
I noticed that bottleneck_block, and by extension dropblock, is used for resnet models with over 50 layers:
https://github.com/tensorflow/tpu/blob/b24729de804fdb751b06467d3dce0637fa652060/models/official/resnet/resnet_model.py#L836-L842

In the paper `dropblock` is mentioned in the following way:
> ImageNet progress has been boosted by innovations in training and regularization approaches. When training models for more epochs, regularization methods such as dropout (Srivastava et al., 2014), label smoothing (Szegedy et al., 2016), stochastic depth (Huang et al., 2016), dropblock (Ghiasi et al., 2018) and data augmentation (Zhang et al., 2017; Yun et al., 2019; Cubuk et al., 2018; 2019) have significantly improved generalization.

and also

> Lee et al. (2020) further improved the ResNet-50 model with additional architectural modifications such as Squeeze-andExcitation (Hu et al., 2018), selective kernel (Li et al., 2019), and anti-alias downsampling (Zhang, 2019), while also using label smoothing, mixup, and dropblock to achieve 81.4% accuracy.

It is not clear from the paper if dropblock is used during training, but it is clearly used in the implementation of resnet in this repo. I'd love to know (for future use) whether or not I should use it as well.
